src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2950,6 +2952,16 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
                 namespace='Microsoft.Network', type='virtualNetwork',
                 name=vnet_peer
             )
+    if loganalytics_workspace_resource_id is not None:
+        loganalytics_workspace_resource_id = loganalytics_workspace_resource_id.strip()
+        if not loganalytics_workspace_resource_id.startswith('/'):
+            loganalytics_workspace_resource_id = '/' + loganalytics_workspace_resource_id
+        if loganalytics_workspace_resource_id.endswith('/'):
+            loganalytics_workspace_resource_id = loganalytics_workspace_resource_id.rstrip('/')
```

Comment:
@jim-minter currently the RP is not doing this logic. We will fail the call if it has a trailing '/'

Reply:
@JackQuincy , we are good on this since the code in this PR takes care of the scenario which you have mentioned. so can you please resolve/close this comment.